pyproject.toml

```diff
@@ -154,7 +154,7 @@ lint = ["_black_check", "_ruff_check", "_ruff_check_nb"]
 _poetry_install_sort_plugin = "poetry self add poetry-plugin-sort"
 _poetry_sort = "poetry sort"
 clean-nbs = "python docs/nbstripout.py"
-format = ["_black_format", "_ruff_format", "_ruff_format_nb", "_poetry_install_sort_plugin", "_poetry_sort"]
```

Comment:
this is a mistake. I removed this only so that it would run on the VM. did not mean to commit it